cypress/integration/channel/message_permalink.js

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/* eslint max-nested-callbacks: ["error", 6] */
+
+import users from '../../fixtures/users.json';
+import * as TIMEOUTS from '../../fixtures/timeouts';
+let testChannel;
```

Comment:
You can create this variable below when it is set it looks like since it's only being used there. Also, if it's not changing use `const`.